### PR TITLE
hack/build: unify operators into one image

### PR DESCRIPTION
**Pull request by fanminshi**

### CODE REVIEW

File: hack/build/docker_push [hunk 1]
```diff
@@ -5,12 +5,12 @@ if ! which docker > /dev/null; then
...
 
 echo "building container..."
-docker build --tag "${IMAGE}" -f hack/build/backup-operator/Dockerfile . 1>/dev/null
+docker build --tag "${IMAGE}" -f hack/build/Dockerfile . 1>/dev/null
 
 # For gcr users, do "gcloud docker -a" to have access.
 
 echo "pushing container..."
-docker push "${IMAGE}" 1>/dev/null
+docker push "${IMAGE}" 1>/dev/null
```
Comment (hongchaodeng) [inline hack/build/docker_push]: new line

**Comment by fanminshi** [comment]
cc/ @hongchaodeng @hasbro17

**Comment by hongchaodeng** [comment]
lgtm

**Comment by fanminshi** [comment]
merge when green.

**Pull request merged**